docs/configuration.rst

```diff
@@ -28,6 +30,8 @@ General Settings
     Twilio_.
   * ``'two_factor.gateways.fake.Fake'``  for development, recording tokens to the
     default logger.
+  * ``'two_factor.gateways.fake.QueryableFake'``  for testing, recording tokens
```

Comment:
If this is merged, the text "Currently two gateways are bundled:" no longer holds.